src/main/java/uk/ac/cam/cl/dtg/teaching/pottery/task/TaskDetails.java

```diff
@@ -21,16 +21,15 @@
 import java.io.File;
 import java.io.IOException;
 import uk.ac.cam.cl.dtg.teaching.pottery.exceptions.InvalidTaskSpecificationException;
-import uk.ac.cam.cl.dtg.teaching.pottery.model.TaskInfo;
 
-public class TaskInfos {
+public class TaskDetails {
```

Comment:
I think the members of this class can now move to be in TaskDetail?

Reply:
Will do.